README.md

```diff
@@ -121,15 +121,15 @@ npm deploy node/name-of-file
 
 👤 **Caleb Nwaizu**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://https://github.com/calebchris000)
+- Twitter: [@twitterhandle](https://twitter.com/calebchris000)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/calebchris000)
 
 👤 **Nwaizu Joshua**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/josh2funny)
+- Twitter: [@twitterhandle](https://twitter.com/josh2funny)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/josh2funny)
 
```

Comment:
- [ ] As suggested in the previous review, Kindly add your own handles and links, I see you tried to do it but you forgot to remove the second author :recycle: